src/main/app/claims/models/claim.ts

```diff
@@ -231,6 +233,9 @@ export class Claim {
     if (this.isPaidInFullLinkEligible()) {
       statuses.push({ status: ClaimStatus.PAID_IN_FULL_LINK_ELIGIBLE })
     }
+    if (this.directionOrder && this.reviewOrder) {
+      statuses.unshift({ status: ClaimStatus.REVIEW_ORDER_REQUESTED })
```

Comment:
out of interest - is there a reason why you used unshift instead of push? does it matter where in the array we add new item?